packages/protocol/contracts/layer1/prover-market/ProverMarket.sol

```diff
@@ -138,7 +144,7 @@ contract ProverMarket is EssentialContract, IProverMarket {
         (address currentProver, uint64 currentFeeInGwei) = _getCurrentProverAndFeeInGwei();
         return currentProver == address(0)
             ? (address(0), 0)
-            : (currentProver, 1 gwei * currentFeeInGwei);
+            : (currentProver, 1 gwei * uint256(currentFeeInGwei));
```

Comment:
there is another occurance of 1 gwei * currentFeeInGwei. How about we define a constant:

uint256 GWEI = uint(1 gwei) and use it in the code.